docs/scenes/get-started/set-up-your-environment/instructions/androidSimulatedDevelopmentBuild.mdx

```diff
@@ -0,0 +1,58 @@
+import { Step } from '~/ui/components/Step';
+import { Terminal } from '~/ui/components/Snippet';
+
+import AndroidStudioInstructions from './_androidStudioInstructions.mdx';
+import AndroidEmulatorInstructions from './_androidEmulatorInstructions.mdx';
+
+## Set up an Android emulator with a development build
+
+<AndroidStudioInstructions />
+<AndroidEmulatorInstructions />
+
+## Create a development build
+
+<Step label="1">
+
+### Install EAS CLI
+
+To build your app, you will need to install EAS CLI. You can do this by running the following command in your terminal:
+
+<Terminal cmd={['$ npm install -g eas-cli']} />
+
+</Step>
+
+<Step label="2">
+
+### Create an Expo account and login
+
+To build your app, you will need to create an Expo account and login to the EAS CLI.
+
+1. [Sign up](https://expo.dev/signup) for an Expo account.
+2. Run the following command in your terminal to login to the EAS CLI:
+   <Terminal cmd={['$ eas login']} />
+
+</Step>
+
+<Step label="3">
+### Configure your project
+
+Run the following command to create an EAS config in your project:
+
+<Terminal cmd={['$ eas build:configure']} />
+
+</Step>
+
+<Step label="4">
+### Create a build
+
+Run the following command to create a development build:
+
+<Terminal cmd={['$ eas build --platform android --profile development']} />
+</Step>
+
+<Step label="5">
+### Install the development build on your emulator
+
+Download the build, then drag and drop the file onto an emulator to install it.
```

Comment:
```suggestion
Download the build, then drag and drop the file onto an emulator to install it.
```

Should we use similar instructions here as of iOS Simulator: http://docs.expo.dev-pr-28511.s3-website-us-east-1.amazonaws.com/get-started/set-up-your-environment/?platform=ios&device=simulated&mode=development-build#install-the-development-build-on-your-simulator? I think this is the easiest path to just press `Y` and install the build using the CLI.